view/Requirements.php

```diff
@@ -1392,9 +1392,6 @@ public function processCombinedFiles() {
 					break;
 				}
 			}
-			if($combinedURL && !$included) {
```

Comment:
I've removed this, because there is a valid case where a combinedFile should be blocked, which is if all of it's source files have been cleared with `->clear`